addons/sourcemod/scripting/vsh/bosses/boss_yeti.sp

```diff
@@ -0,0 +1,179 @@
+#define YETI_MODEL "models/player/kirillian/boss/yeti_modded.mdl"
+#define YETI_THEME "ui/gamestartup27.mp3"
+
+static char g_strYetiRoundStart[][] =  {
+	"ambient_mp3/lair/animal_call_yeti1.mp3", 
+	"ambient_mp3/lair/animal_call_yeti2.mp3", 
+	"ambient_mp3/lair/animal_call_yeti3.mp3", 
+	"ambient_mp3/lair/animal_call_yeti4.mp3", 
+	"ambient_mp3/lair/animal_call_yeti5.mp3", 
+	"ambient_mp3/lair/animal_call_yeti6.mp3", 
+	"ambient_mp3/lair/animal_call_yeti7.mp3"
+};
+
+static char g_strYetiWin[][] =  {
+	"player/taunt_yeti_roar_first.wav"
+};
+
+static char g_strYetiLose[][] =  {
+	"player/taunt_yeti_roar_second.wav"
+};
+
+static char g_strYetiFreeze[][] =  {
+	"player/taunt_yeti_appear_snow.wav", 
+	"weapons/icicle_freeze_victim_01.wav"
+};
+
+static char g_strYetiGroundSlam[][] =  {
+	"player/taunt_yeti_land.wav"
+};
+
+static char g_strYetiKill[][] =  {
+	"player/taunt_yeti_roar_beginning.wav"
+};
+
+static char g_strYetiLastMan[][] =  {
+	"ambient_mp3/lair/animal_call_yeti2.mp3", 
+};
+
+static char g_strYetiBackStabbed[][] =  {
+	"player/taunt_yeti_roar_second.wav"
+};
+
+static char g_strYetiVoice[][] =  {
+	"player/taunt_yeti_roar_first.wav", 
+	"player/taunt_yeti_roar_second.wav"
+};
+
+static char g_strYetiFootsteps[][] =  {
+	"player/footsteps/giant1.wav", 
+	"player/footsteps/giant2.wav"
+};
+
+methodmap CYeti < SaxtonHaleBase
+{
+	public CYeti(CYeti boss)
+	{
+		// TODO: Abilities
+		boss.CallFunction("CreateAbility", "CBraveJump");
+		boss.CallFunction("CreateAbility", "CGroundPound");
+		//boss.CallFunction("CreateAbility", "CRageFreeze");
+		
+		boss.iBaseHealth = 800;
+		boss.iHealthPerPlayer = 850;
+		boss.nClass = TFClass_Heavy;
+		boss.iMaxRageDamage = 2500;
+	}
+	
+	public void GetBossName(char[] sName, int length)
+	{
+		strcopy(sName, length, "Last Yeti");
+	}
+	
+	public void GetBossInfo(char[] sInfo, int length)
+	{
+		StrCat(sInfo, length, "\nHealth: High");
+		StrCat(sInfo, length, "\n ");
+		StrCat(sInfo, length, "\nAbilities");
+		StrCat(sInfo, length, "\n- Brave Jump");
+		StrCat(sInfo, length, "\n- Ground Slam");
+		StrCat(sInfo, length, "\n ");
+		StrCat(sInfo, length, "\nRage");
+		StrCat(sInfo, length, "\n- Slows every player around the yeti for 3 seconds with a freezing aura");
+		StrCat(sInfo, length, "\n- Then completely freezes everyone affected by the aura for 5 seconds");
+		StrCat(sInfo, length, "\n- 200%% Rage: Extends duration to 10 seconds");
+	}
+	
+	public void OnSpawn()
+	{
+		char attribs[128];
+		Format(attribs, sizeof(attribs), "2 ; 2.80 ; 252 ; 0.5 ; 259 ; 1.0 ; 214 ; %d", GetRandomInt(9999, 99999));
+		int iWeapon = this.CallFunction("CreateWeapon", 195, NULL_STRING, 100, TFQual_Strange, attribs);
```

Comment:
You want strange "hales own" stat for yeti's weapon? I feel that should be kept to hale only.

Reply:
Good point. Which rank should we go with? "Legendary" would fit well.

Reply:
Either that or "Rage-Inducing" would work